scripts/distribution/macOS-package/NodeConfigurationInstallerPlugin/NodeConfigurationInstallerPlugin/MyInstallerPane.m

```diff
@@ -67,26 +68,31 @@ - (void) saveConfigurationToDisk {
     os_log(customLog, "Create file: %@", configFilePath);
     BOOL createdFile = [[NSFileManager defaultManager] createFileAtPath:configFilePath contents:nil attributes:nil];
     os_log(customLog, "File-creation result: %d", createdFile);
-    
+
+    // Escape characters with special meaning in XML.
+    // The method is only available on CFStringRef, so: convert to CFStringRef, escape XML, convert back to NSString.
+    NSString *escapedMainnetName = (__bridge NSString*)CFXMLCreateStringByEscapingEntities(CFAllocatorGetDefault(), (__bridge CFStringRef)[_oMainnetNodeName stringValue], NULL);
```

Comment:
The documentation of this method states
```
Specifies the entities to be replaced. Dictionary keys should be the entity names (for example, “para” for &para;), and the values should be CFString objects containing the expansion. Pass NULL to indicate no entities other than the standard five.
```

What are the "standard five"?

Reply:
My guess would be something along the lines of: T = String | Number | Bool | T | [T]   ....I cannot find a source though.

Reply:
According to the [six-year-old source](https://github.com/opensource-apple/CF/blob/3cc41a76b1491f50813e28a4ec09954ffa359e6f/CFXMLParser.c#L1902), the standard five are: `&` `<` `>` `\` `"`
However, I can see that `'` is also escaped, so perhaps they've expanded the list of standard characters to be escaped in a more recent version.